docs/physics/tardisgamma/opacities.ipynb

```diff
@@ -0,0 +1,399 @@
+{
```

Comment:
<p>please add a link to what equation it is in the papers. Also please add units to the formula</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2876/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>